plenoptic/synthesize/Synthesis.py

```diff
@@ -479,11 +490,11 @@ def _finalize_stored_progress(self):
 
     @abc.abstractmethod
     def synthesize(self, seed=0, max_iter=100, learning_rate=1, scheduler=True, optimizer='Adam',
-                   clamper=RangeClamper((0, 1)), clamp_each_iter=True, store_progress=False,
+                   optimizer_kwargs={}, swa=False, swa_kwargs={}, clamper=RangeClamper((0, 1)),
```

Comment:
todo: We had meeting Jan8 talking about in the future super-classing this into a simpler form.

Reply:
Yup, issue #75. Let's leave this stuff for now, but it will be good to simplify all this.